src/Utils.js

```diff
@@ -149,7 +149,8 @@ export function AND(fns) {
   };
 }
 
-export function coercePropValue(propValue) {
+export function coercePropValue(propValue, propName) {
```

Comment:
every other internal function for this has had the signature of `fn(key, value) {}` maybe swapping these two for consistency sake?